src/includes/Helpers/PaymentHelper.php

```diff
@@ -532,7 +537,7 @@ protected function add_product_data( $item ) {
 			'line_price'        => $this->tool_helper->alma_price_to_cents( $item->get_total() ),
 			'categories'        => $categories,
 			'url'               => $product->get_permalink(),
-			'picture_url'       => wp_get_attachment_url( $product->get_image_id() ),
+			'picture_url'       => $picture_url,
```

Comment:
I think a ternary would be more suitable For php5.6 compatibility a full ternary :
`wp_get_attachment_url( $product->get_image_id() ) ? wp_get_attachment_url( $product->get_image_id() ) : '';`
for php 7 you can write : 
`wp_get_attachment_url( $product->get_image_id() ) ?: ''`